packages/payment-integration-test-framework/src/fixture/pageObject/playwright/PlaywrightHelper.ts

```diff
@@ -24,6 +25,7 @@ export class PlaywrightHelper {
         this.polly = new PollyObject(this.mode);
         this.server = new ServerSideRender();
         this.storeUrl = getStoreUrl();
+        this.srcPath = './packages/payment-integration-test-framework/src';
```

Comment:
Do we need to do this in the constructor?

Reply:
`playwright` helper uses this prop three times for locating support files, such as HTML template.

What do you think about the package name, `payment-integration-test-framework`? I wanted to put it in line with `payment-integration-api`, but made it quite long😂.

Reply:
Should be able to declare this variable after line 19, just need to declare it as public.

Reply:
The package name seems fine to me, the `api` keyword doesn't feel right here since its not the api package. Its named correctly I think as of now :D